documentation/style-guide.rst

```diff
@@ -103,11 +103,16 @@ Unix
 Use simple language
 ===================
 
-Avoid esoteric phrasing where possible.  Our audience is world-wide and may not
-be native English speakers.
-
-Don't use Latin abbreviations like "e.g." or "i.e." where English words will do,
-such as "for example" or "that is."
+The documentation has a world-wide audience of varying English proficiency.
+To make the documentation more accessible and inclusive, the following should be
+avoided:
+
+- Esoteric phrasing.
```

Comment:
Ironically "Esoteric phrasing" could arguably fit its own definition, even though these rules might not necessarily apply to the devguide itself. 🙃